ompi/mca/io/romio341/configure.m4

```diff
@@ -99,14 +99,15 @@ AC_DEFUN([MCA_ompi_io_romio341_CONFIG],[
                           . 3rd-party/romio341/localdefs
                           io_romio341_LIBS="$LIBS"
                           LIBS="$io_romio341_save_LIBS"
-                          OPAL_3RDPARTY_SUBDIRS="$OPAL_3RDPARTY_SUBDIRS romio341"
                           OPAL_3RDPARTY_DIST_SUBDIRS="$OPAL_3RDPARTY_DIST_SUBDIRS romio341"
 
                           echo "ROMIO distribution configured successfully"
```

Comment:
If I'm reading this right, and I may not be, the output from running configure will be something like:

```
ROMIO distribution configured successfully
Checking for stdatomic.h: no
ROMIO distribution did not configure successfully
```